### PR TITLE
kapow-punch-clock: Add version 1.5.10

### PR DESCRIPTION
**Pull request by zStruCat**
Kapow is a portable program to help tracking spent time
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/kapow-punch-clock.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.5.10",
+    "description": "A portable program to help tracking spent time",
+    "homepage": "https://gottcode.org/kapow/",
+    "license": "GPL-3.0-or-later",
+    "url": "https://gottcode.org/kapow/download/?os=windowsportable#/kapow-portable.zip",
+    "hash": "0ca9398354cb457baf80e01de90386384d6e4eb0666ba615f124f4b9af6a6f10",
+    "extract_to": "",
```
Comment (rashil2000) [inline bucket/kapow-punch-clock.json:8]:
```suggestion
```

File: bucket/kapow-punch-clock.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        [
+            "Kapow.exe",
+            "KapowClock"
+        ]
+    ],
+    "persist": "data",
+    "checkver": {
+        "url": "https://gottcode.org/kapow/",
+        "regex": ">Version ([\\d\\.]+)<"
+    },
```
Comment (rashil2000) [inline bucket/kapow-punch-clock.json:19]:
```suggestion
    "checkver": ">Version ([\\d\\.]+)<",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
This is 64bit app. Use architecture block.

**Comment by zStruCat** [comment]
/verify
manifest changed to architecture.64bits

File: bucket/kapow-punch-clock.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.5.10",
+    "description": "A portable program to help tracking spent time",
```
Comment (rashil2000) [inline bucket/kapow-punch-clock.json:3]:
```suggestion
    "description": "A punch clock program to help in tracking spent time",
```

**Comment by zStruCat** [comment]
> Rename app to just kapow.json

I used the name deliberately as there're other [program](https://github.com/BBVA/kapow) named as kapow. And i don't want to take risk.

**Comment by rashil2000** [comment]
> > Rename app to just kapow.json
> 
> I used the name deliberately as there're other [program](https://github.com/BBVA/kapow) named as kapow. And i don't want to take risk.

Ah, didn't know about it. Thanks for letting me know.

**Pull request merged**